docs/whats-new/dotnet-aspire-9.1.md

```diff
@@ -139,9 +143,41 @@ This release also focused on improving various [Azure integrations](../azure/int
 - Emulator support—`RunAsEmulator` APIs were added to the following integrations:
   - [Add Azure Service Bus emulator resource](../messaging/azure-service-bus-integration.md#add-azure-service-bus-emulator-resource).
   - [Add Azure Event Hubs emulator resource](../messaging/azure-event-hubs-integration.md#add-azure-event-hubs-emulator-resource).
-- It's simpler to connect to existing Azure resources in the app host.
 - Experimental support for configuring custom domains in Azure Container Apps (ACA) was added.
 
+#### Leverage existing Azure resources
```

Comment:
We have added all of this content for 9.1, here:

- https://review.learn.microsoft.com/en-us/dotnet/aspire/azure/integrations-overview?branch=pr-en-us-2566&tabs=dotnet-cli#use-existing-azure-resources

You might consider borrowing from that and then linking to it for more detail?

Reply:
You should sync your branch. I expanded on all the Azure integration 9.1 features in [Enhance Integrations What's New in 9.1 (dotnet/docs-aspire#2657)](https://github.com/dotnet/docs-aspire/pull/2657)